src/freenet/clients/http/PproxyToadlet.java

```diff
@@ -533,7 +533,7 @@ private void showPluginList(ToadletContext ctx, PluginManager pm, HTMLNode conte
 					pluginRow.addChild("td");
 				} else {
 					if (pi.isPproxyPlugin()) {
-						HTMLNode visitForm = pluginRow.addChild("td").addChild("form", new String[] { "method", "action", "target" }, new String[] { "get", pi.getPluginClassName(), "_blank" });
+						HTMLNode visitForm = pluginRow.addChild("td").addChild("form", new String[] { "method", "action", "target", "rel" }, new String[] { "get", pi.getPluginClassName(), "_blank", "noreferrer noopener" });
```

Comment:
Do I understand it correctly that this leaves out the referrer when accessing a plugin page from a freesite?

If yes: Why do you consider that as problematic?